pages/common/fossa.md

```diff
@@ -0,0 +1,24 @@
+# fossa
+
+> Fossa CLI - Fast, portable and reliable dependency analysis.
```

Comment:
We typically don't use adjectives like "fast" and "reliable". Please stick to an objective description, and perhaps include a bit more details about what this does (what does "dependency analysis" mean in practice? what sort of outcome can one get from this tool?)

Reply:
I have changed this. 

The outcome is to get reports about all dependencies licenses and related issues to that.